website/content/en/docs/advanced-topics/scorecard/kuttl-tests.md

```diff
@@ -4,160 +4,224 @@ weight: 50
-
-## Configuring kuttl Tests in the Scorecard Configuration
+If you look at scorecard's [configuration file][scorecard-config] `bundle/tests/scorecard/config.yaml`,
+you'll notice nothing that refers to the scorecard-test-kuttl image described above. This is by design:
+kuttl tests should be enabled only once your operator is ready for in-cluster testing. Therefore the
+following patch specified in `config/scorecard/kustomization.yaml` was commented out at first:
 
-In the scorecard configuration file, you might have the following
-definition of what the selector `suite=kuttlsuite` will translate to:
 ```yaml
-stages:
-- tests:
-  - image: quay.io/operator-framework/scorecard-test-kuttl:dev
-    labels:
-      suite: kuttlsuite
-      test: kuttltest1
+patchesJson6902:
+...
+#- path: patches/kuttl.config.yaml
+#  target:
+#    group: scorecard.operatorframework.io
+#    version: v1alpha3
+#    kind: Configuration
+#    name: config
 ```
 
-This test configuration will execute the scorecard-test-kuttl
-image which executes kuttl.  The kuttl output is translated
-into scorecard compliant output which is displayed back to the
-end user along with any other test results.
+Now that you've written a robust test suite and your operator is ready for testing, uncomment the above
+patch and regenerate your bundle:
 
-With the above kuttl test configuration, you can execute that
-kuttl test using scorecard as follows:
-```bash
-operator-sdk scorecard <bundle_dir_or_image> --selector=suite=kuttlsuite 
+```sh
+make bundle IMG=<operator image>
 ```
 
-## Defining kuttl Specific Configuration Options
-
-The [kuttl configuration file][kuttl_yaml] is documented within the
-kuttl project.
-
-An example of the kuttl configuration file is as follows:
+If you inspect `bundle/tests/scorecard/config.yaml` you should now see a kuttl test stage:
 
 ```yaml
-apiVersion: kudo.dev/v1beta1
-kind: TestSuite
-parallel: 4
-timeout: 120
-startControlPlane: false
+apiVersion: scorecard.operatorframework.io/v1alpha3
+kind: Configuration
+metadata:
+  name: config
+stages:
+- parallel: true
+  tests:
+    ...
+- parallel: true
+  tests:
+  - image: quay.io/operator-framework/scorecard-test-kuttl:v1.2.0
+    labels:
+      suite: kuttl
 ```
 
-The important fields to note here are:
-
- * `startControlPlane` - Set to false since scorecard assumes it is running
-within a control plane already.
-
-Other kuttl configurations settings are available for more advanced kuttl
-use cases.  See [kuttl configuration][kuttl_configuration] for more details on kuttl configuration.
-
-### kuttl Tests Explained
-
-The kuttl test tool looks for tests to execute within the bundle 
-following a naming convention as follows:
+All that is left to do is to copy your tests to `bundle/tests/scorecard` and run the `scorecard` command
+(assuming the `memcached-operator-system` namespace has Memcached CRUD permissions):
+
+```console
+$ cp -r test/kuttl bundle/tests/scorecard
```

Comment:
Is it missing a dot? Do we need to copy the tests from the bundle dir to the root dir?

Reply:
Test dependencies should never be part of the operator bundle image. This is discussed above:

>Typically you should not bundle test dependencies with your production manifests, so it is currently recommended that you simply copy your kuttl manifests into `bundle/` while testing.

Eventually there will be an test bundle image spec designed.

Reply:
See: https://github.com/operator-framework/operator-sdk/pull/4329/files#r586850978